### DISCUSSION TITLE
How to remove new onscreen items "owned attribute", "similar note", and note directory.

### DISCUSSION THREAD

**Discussion by Tandabock** [TriliumNext/Trilium#1468] [Q&A]
I upgraded from version 0.40.1 to 0.45.5, and three additional items are now showing up onscreen that I have no need of and want to remove.
1) "owned attributes" on the top line of the note. 
2) "similar note" on the bottom line of the note. 
3) Additional note directory line at the top of the tree list on the left, showing the current open note in a directory format: "root/note name 1/note name 2", or at lease the few letters of the directory that can fit in that small space.

I don't want to just hide the results of what these items find, I want the entire line item along with the words "owned attribute" and "similar notes" and the space used for the directory line removed. 

If the only way to remove these items is to go back to version 0.40.1, then I will do that. 

Just a note about general design principals. Screen real estate is valuable. Adding items that permanently show onscreen should be absolutely necessary, not something that only some users want. Any added onscreen item should have the option to be removed in settings.

Besides this though, Trilium is about the best organizational program I've found.

**Comment by zadam** [comment]
> I don't want to just hide the results of what these items find, I want the entire line item along with the words "owned attribute" and "similar notes" and the space used for the directory line removed.

You can collapse the content so that it takes minimal amount of space.

If you want them really removed / hidden then you need to use custom CSS (using `#appCss` label) which will hide these elements. But I wouldn't do that at least in the case of attributes since then you lose the only way to edit the attributes.
